api/node/package.json

```diff
@@ -1,25 +1,49 @@
 {
-    "name": "slint-ui",
-    "version": "1.3.0",
-    "homepage": "https://github.com/slint-ui/slint",
-    "license": "SEE LICENSE IN LICENSE.md",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/slint-ui/slint"
-    },
-    "main": "dist/index.js",
-    "types": "dist/index.d.ts",
-    "dependencies": {
-        "@types/node": "^14.11.11",
-        "neon-cli": "^0.4",
-        "typescript": "^4.0.3"
-    },
-    "scripts": {
-        "install": "neon build --release && tsc",
-        "build": "tsc",
-        "docs": "typedoc --hideGenerator --readme cover.md lib/index.ts"
-    },
-    "devDependencies": {
-        "typedoc": "^0.19.2"
+  "name": "slint-ui",
+  "version": "1.3.0",
+  "main": "index.js",
+  "types": "index.d.ts",
+  "license": "GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial",
+  "napi": {
+    "name": "slint-ui"
+  },
+  "devDependencies": {
+    "@napi-rs/cli": "^2.15.2",
+    "@swc-node/register": "^1.5.5",
+    "@swc/core": "^1.3.32",
+    "@types/node": "^20.8.6",
+    "ava": "^5.3.0",
+    "esbuild": "^0.14.54",
+    "jimp": "^0.22.8",
+    "typedoc": "^0.25.2"
+  },
+  "engines": {
+    "node": ">= 10"
+  },
+  "scripts": {
+    "artifacts": "napi artifacts",
+    "compile": "esbuild index.ts --bundle --external:*.node --format=cjs --platform=node --outfile=index.js",
+    "build": "napi build --platform --release --js rust-module.js --dts rust-module.d.ts && npm run compile",
+    "build:debug": "napi build --platform --js rust-module.js --dts rust-module.d.ts && npm run compile && npm run syntax_check",
+    "install": "npm run build",
+    "docs": "npm run build && typedoc --hideGenerator --treatWarningsAsErrors --readme cover.md index.ts",
+    "prepublishOnly": "napi prepublish -t npm",
+    "test": "ava",
+    "syntax_check": "tsc -noEmit index.ts",
+    "universal": "napi universal",
+    "version": "1.3.0"
```

Comment:
What is the purpose of this field in the "scripts" object?

Reply:
I think for our use case we only need  `test`

Reply:
I removed that fields, because everything works without it. And we should add fields if we need it.